packages/oauth/oauth-provider-api/src/api-endpoints.ts

```diff
@@ -13,12 +13,12 @@ export type ApiEndpoints = {
   '/sign-up': {
     method: 'POST'
     input: SignUpInput
-    output: { account: Account }
+    output: { account: Account; token?: string }
```

Comment:
Since there are lots of "tokens" floating around the auth flows, I wonder if it may be nice to give these a more specific name to indicate what they are for / what their lifetime is.